src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/Table/filters_row.html.twig

```diff
@@ -26,14 +26,17 @@
 <tr class="column-filters {% if 0 == grid.data.records_total and grid.filters is empty and grid.actions.bulk|length < 0 %}d-none{% endif %}">
   {% for column in grid.columns %}
     <td>
+      {% set cssClassFormWidget = ''  %}
       {% if loop.first and grid.actions.bulk|length > 0 %}
+        {% set cssClassFormWidget = ' d-inline-block' %}
```

Comment:
@eternoendless I know what do you think seeing . this, should we change this class ?

Reply:
So I'm not sure I get it, if you have bulk actions AND it's in the first column, then the filters displayed in this column should have an `inline` display?

Reply:
I actually wonder, isn't it simpler if we always have inline blocks?

Reply:
The correct thing would be to have display: inline-block; on a proper selector css-sided, that's it so it would be something like 
```suggestion
        {% set cssClassFormWidget = ' grid-column-checkbox' %}
```

Reply:
The correct thing would be to have display: inline-block; on a proper selector css-sided, that's it so it would be something like 
```suggestion
        {% set cssClassFormWidget = ' grid-column-checkbox' %}
```

and in the css of the grid :
```
  .grid-column-checkbox {
    display: inline-block;
  }
```

and if it not display block in all cases, there should be a modifier class added using a condition like `.grid-column-checkbox-inline`

Reply:
I don't understand why we can't use classes from PrestaShop UI Kit based on the framework Twitter Bootstrap. It would be recreate classes because we don't want to use Symfony framework.

Reply:
@NeOMakinG any explanation to move the PR forward?

Reply:
@Progi1984 "helpers classes" are against the base of CSS and webcomponents principles, why even using `d-inline-block` classes as it is the same as using a `style=""` attribute. Creating a class to use only one CSS property is too weird, if we follow that idea, then let's create a class for every properties and never touch stylesheets anymore.

To be honest, that's just laziness. I'm actually working on doing returns from the product team on View Order pages and theses classes make it hard to customize almost everything because they are often using !important and they make it really harder.

For this time as I said on some others PR's, let's stay like this, but before next migrations, we have to speak of a proper way to integrate design's using good webcomponents guidelines and workflow.

@eternoendless @PierreRambaud help...

Reply:
I totally agree what @NeOMakinG said :+1: 
Using these kinds of "classes" is a bad practice and make the customisation harder to be done!

I'm 100% sure, if our front-end developer says it's a bad practice and these classes shouldn't be used, we must trust him! He's the one here who's working with CSS & HTML, we shouldn't argue and listen him!

Reply:
I agree with @NeOMakinG on this. Helper classes are useful for quick prototyping but are actually antipatterns, because they violate the principle of separating style from semantics, which is the reason behind CSS files being a thing.

If changing the style of an element requires you to change the object's markup, **_you're doing it wrong_**.

Reply:
I do it, but always not agree